src/main/java/com/getindata/connectors/http/table/HttpDynamicSinkConfig.java

```diff
@@ -0,0 +1,50 @@
+package com.getindata.connectors.http.table;
+
+import lombok.*;
+import lombok.extern.slf4j.Slf4j;
+
+import java.io.Serializable;
+
+/**
+ * The configuration for {@link HttpDynamicSink}, containing the data specific for the
+ * {@link com.getindata.connectors.http.sink.HttpSink}.
+ */
+@Builder
+@Data
+@RequiredArgsConstructor
+@Slf4j
+public class HttpDynamicSinkConfig implements Serializable {
```

Comment:
Instead POJO class could we use something like this one here:
https://github.com/delta-io/connectors/blob/master/flink/src/main/java/io/delta/flink/source/internal/DeltaSourceConfiguration.java

Then we will use `HttpDynamicSinkConnectorOptions` as a source of truth for available options.

DeltaSourceConfiguration.java uses [DeltaSourceOptions.java](https://github.com/delta-io/connectors/blob/master/flink/src/main/java/io/delta/flink/source/internal/DeltaSourceOptions.java) but we dont need to have our own class for options.